src/Client/src/api/videos/api.ts

```diff
@@ -80,10 +80,13 @@ export function useUpdateVideo(id: string) {
     (data: UpdateVideoFormValues) => {
       const formData = new FormData();
 
-      formData.append("name", data.name);
+      formData.append("name", data.name ?? "");
       formData.append("description", data.description ?? "");
       
-    return  axios.put(`${API_BASE_URL}/video/${id}`, formData );
+    return  axios.put(`${API_BASE_URL}/video/${id}`, formData , {
+      headers: { "Content-Type": "multipart/form-data" },
+      withCredentials: true,
+    });
     },
```

Comment:
check useCreateVideoComment() hook in https://github.com/kristupas-g/videoteka-app/blob/main/src/Client/src/api/videoComments/api.ts. dont forget to add invalidateQueries("videos") - it will refresh the queries which have the "videos" key once you update a video. its required because otherwise old data would be shown.

also use axiosInstance instead of axios. you wont need headers and withcredentials then. :)